server/api/orders.js

```diff
@@ -1,6 +1,7 @@
 const ordersRouter = require('express').Router();
```

Comment:
your orders router doesn't seem to have the auth updates. Might need to merge the master into this branch before committing

Reply:
these 2 new routes are just for getting order_items and creating new ones, i dont think that we will need to authorise the user id